src/mapping-v3-core.ts

```diff
@@ -552,6 +555,110 @@ export function handleMinterUpdated(event: MinterUpdated): void {
   contractEntity.save();
 }
 
+// Handle artist proposed address and splits updates
+// This is an event indicating that the artist has proposed a new set of
+// addresses and splits for a project.
+export function handleProposedArtistAddressesAndSplits(
```

Comment:
Totally fine to leave the code as is but we could reduce the lines of code here by not clearing the store between each test. If you decide to go this route I'd suggest wrapping everything here in a describe block to give it some explicit logical grouping.

Also, can you add a couple tests for the early return cases.

Reply:
@ryley-o this was meant to be in the test file oops!

Reply:
got it 👍 good call, updated with describe block and a few tests for the early return cases.

Reply:
(just to note, I kept the`clearStore` in a BeforeEach, just to help me keep things clean between the different tests)